client/src/index.jsx

```diff
@@ -164,29 +211,6 @@ const person = {
 	avatarUrl:"fdsfsdf"
```

Comment:
Can we change this to not be a 404? Seeing the red in the console makes me thing something is broken when its not

Reply:
Removed the image URL fetch